docs/ml-features.md

```diff
@@ -1558,6 +1558,15 @@ for more details on the API.
 
 {% include_example java/org/apache/spark/examples/ml/JavaBucketedRandomProjectionLSHExample.java %}
 </div>
+
+<div data-lang="python" markdown="1">
+
+Refer to the [BucketedRandomProjectionLSH Python docs](api/python/pyspark.ml.html#pyspark.ml.feature.BucketedRandomProjectionLSH)
+for more details on the API.
+
+{% include_example python/ml/bucketed_random_projection_lsh.py %}
```

Comment:
These are not correct and the docs don't build because of it. In the future, can you check that the docs build when you make changes?

`cd docs; SKIP_API=1 jekyll serve --watch`

More detailed instructions [here](https://github.com/apache/spark/tree/master/docs). Also you can build the python docs by `cd python/docs; make html`